<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Solicitudes</class>
 <widget class="QMainWindow" name="Solicitudes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 229, 158);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grbUsuarios">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>1121</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka&quot;;</string>
    </property>
    <property name="title">
     <string>USUARIOS</string>
    </property>
    <widget class="QTableWidget" name="tlbUs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>1091</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbRec">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>621</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka&quot;;</string>
    </property>
    <property name="title">
     <string>SOLICITUDES RECIBIDAS</string>
    </property>
    <widget class="QTableWidget" name="tlbRec">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>591</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbEnv">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>491</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka&quot;;</string>
    </property>
    <property name="title">
     <string>SOLICITUDES ENVIADAS</string>
    </property>
    <widget class="QTableWidget" name="tlbEnv">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>471</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>640</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
